### PR TITLE
Add rulesets for sites by Mirco Neubert

### PR DESCRIPTION
**Pull request by rugk**
Dynatech.de
Dynavision.de
Scieneo.de

### CODE REVIEW

File: src/chrome/content/rules/Dynavision.de.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	    - Dynatech.de
+		
+-->
+<ruleset name="Dynavision.de">
+    <target host="www.dynavision.de" />
+
+    <securecookie host="^www\.dynavision\.de" name=".+" />
+
+ 	<!-- Redirect broken domain -->
+    <rule from="^http(s)?://dynavision.de"
```
Comment (J0WI) [inline src/chrome/content/rules/Dynavision.de.xml]: You don't need to rewrite the https version

**Comment by J0WI** [comment]
You don't need to rewrite the https version

**Comment by rugk** [comment]
[**@J0WI**](https://github.com/J0WI) commented on [16. Dez. 2015, 23:38 MEZ](https://github.com/EFForg/https-everywhere/pull/3459#issuecomment-165276071 "2015-12-16T22:38:57Z - Replied by Github Comment Enhancer"):
> You don't need to rewrite the https version

Really? But accessing https://dynatech.de e.g. causes in an error. So should I only redirect the http version?

**Comment by J0WI** [comment]
Yes, because accessing a broken https site is a less common scenario.

**Comment by rugk** [comment]
Okay, changed.

**Comment by J0WI** [comment]
You need to add the ^ to the list of target hosts. This should also make the test url obsolete.

**Comment by rugk** [comment]
To the list of target hosts? In no ruleset I have ever seen was this done, especially as I doubt that the target hosts attribute supports regular expressions.

**Comment by J0WI** [comment]
Yes, but regular expressions are not very fail-safe.
Have a look at the last example on https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md

**Comment by rugk** [comment]
[**@J0WI**](https://github.com/J0WI) commented on [18. Dez. 2015, 22:47 MEZ](https://github.com/EFForg/https-everywhere/pull/3459#issuecomment-165904873 "2015-12-18T21:47:30Z - Replied by Github Comment Enhancer"):
> Have a look at the last example on [https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md)

Okay:
`  <target host="whatwg.org" />`

That's how I did it. Without a caret `^` here.
And as the last Travic CI test worked I don't know what you suggest to change.

**Comment by J0WI** [comment]
Simply add 
```<target host="dynatech.de" />```
instead of
```<test url="http://dynatech.de/" />```

File: src/chrome/content/rules/Dynavision.de.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!--
+    Unfunctional subdomains:
+	    - $
+
+    More rulesets for sites by Mirco Neubert:
+        - Dynavision.de
```
Comment (J0WI) [inline src/chrome/content/rules/Dynavision.de.xml]: Shouldn't this be ```Scieneo.de```

File: src/chrome/content/rules/Scieneo.de.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<!--
+    Nonfunctional subdomains:
+	    - $
+
+    More rulesets for sites by Mirco Neubert:
+	    - Dynatech.de
+	    - Scieneo.de
```
Comment (J0WI) [inline src/chrome/content/rules/Scieneo.de.xml]: And here ```Dynavision.de```

Comment (J0WI) [inline src/chrome/content/rules/Scieneo.de.xml]: And they are easier to find when you use the file name (e.g. ```Dynavision.de.xml ```) ;)

**Comment by rugk** [comment]
Ah now I understand your issue. Actually I cannot do this as it causes errors.
So the real problem are the test URLs. They are wrong.
I'll change that.

File: src/chrome/content/rules/Scieneo.de.xml (hunk 3 above)
Comment (rugk) [inline src/chrome/content/rules/Scieneo.de.xml]: Yes, thanks. But AFAIK usually the ruleset names (without file extension) are used.

File: src/chrome/content/rules/Dynavision.de.xml (hunk 2 above)
Comment (rugk) [inline src/chrome/content/rules/Dynavision.de.xml]: Yes, thnks. Copy and paste issue...

**Comment by rugk** [comment]
A no. Actually you're right. Of course I have to include the hosts as targets. As I have included a redirect rule this is obviously possible. Sorry, this was an error in reasoning.

**Comment by J0WI** [comment]
lgtm

**Comment by rugk** [comment]
Finally. :wink:

**Comment by J0WI** [comment]
cc @semenko @loveithateit @fuglede

**Pull request merged**